xcodeproj/internal/bazel_integration_files/xcodeproj.bazelrc

```diff
@@ -0,0 +1,20 @@
+# `rules_xcodeproj` is the base config of `rules_xcodeproj_*` configs.
+# Add to this config if you want something to apply to all builds (which should
+# be most settings), as the various configs (e.g. `rules_xcodeproj_indexbuild`)
+# don't inherit from `rules_xcodeproj_build`. This allows
+# `rules_xcodeproj_build` to strictly apply to the normal build.
+build:rules_xcodeproj --verbose_failures
+
+# `rules_xcodeproj_generator` is used when generating a project.
+build:rules_xcodeproj_generator --config=rules_xcodeproj
+build:rules_xcodeproj_generator --show_result=0
+build:rules_xcodeproj_generator --ui_event_filters=-INFO,-WARNING
+
+# `rules_xcodeproj_build` is used when doing a normal build
+build:rules_xcodeproj_build --config=rules_xcodeproj
+
+# `rules_xcodeproj_build` is used when doing an Index build
```

Comment:
```suggestion
# `rules_xcodeproj_indexbuild` is used when doing an Index build
```

Reply:
Thanks. In the future though, a single comment on files in `bazel_integration_files`, or one of the generated files, is good enough, since changing one will change all of the others.